android/app/src/main/java/au/com/robin/sms/MainActivity.kt

```diff
@@ -0,0 +1,69 @@
+package au.com.robin.sms
+
+import android.Manifest
+import android.content.Intent
+import android.content.pm.PackageManager
+import android.os.Bundle
+import android.util.Log
+import android.widget.Toast
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.app.ActivityCompat
+import androidx.core.content.ContextCompat
+import au.com.robin.sms.databinding.ActivityMainBinding
+
+class MainActivity : AppCompatActivity() {
+    private lateinit var smsInterfaceBinding: ActivityMainBinding
+    private val PERMISSION_REQUEST_CODE = 1
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        smsInterfaceBinding = ActivityMainBinding.inflate(layoutInflater)
+        val view = smsInterfaceBinding.root
+        setContentView(view)
+
+        smsInterfaceBinding.btSend.setOnClickListener {
+            val phoneNo = smsInterfaceBinding.etPhoneNo.text?.toString()
+            val message = smsInterfaceBinding.etMessage.text?.toString()
+
+            // Check for permission
+            if (checkPermission()) {
+                // create an intent to let client choose which sim card slot
+                // for now, we use sim card 2. Normally, user will be shown options to choose from.
+                val intent = Intent()
+                intent.putExtra("slot", 1)
```

Comment:
Explain what does putting 1 in slot intent means

Reply:
Do you want to have an option where user can pick which sim card if they are multi sim card users?

Reply:
Yes but maybe not this PR.

Reply:
Okay, noted!